package.json

```diff
@@ -1,51 +1,39 @@
 {
-  "name": "@node-saml/node-saml",
-  "version": "4.0.3",
-  "description": "SAML 2.0 implementation for Node.js",
+  "name": "@foxden/node-saml",
+  "version": "4.0.4",
```

Comment:
Version number doesn't matter since we're technically creating a "new" package. You can set this to 1.0.0.